Formula/z.rb

```diff
@@ -1,8 +1,9 @@
 class Z < Formula
   desc "Tracks most-used directories to make cd smarter"
   homepage "https://github.com/rupa/z"
-  url "https://github.com/rupa/z/archive/v1.9.tar.gz"
-  sha256 "e2860e4f65770e02297ca4ca08ec1ee623a658bd9cc1acddbbe5ad22e1de70a7"
+  url "https://github.com/rupa/z.git",
+    :tag => "v1.11",
```

Comment:
1.9 is still indicated as “latest release”

Reply:
Opened https://github.com/rupa/z/issues/213 asking about this...

Reply:
See https://github.com/rupa/z/issues/205.